libs/components/lookup/src/lib/modules/search/search.component.scss

```diff
@@ -89,16 +68,41 @@ sky-search {
     .sky-search-input,
     .sky-search-btn-apply,
     .sky-search-btn-clear {
-      font-size: $sky-font-size-base;
+      font-size: var(
+        --sky-override-search-mobile-font-size,
+        var(--sky-font-size-body-m)
+      );
     }
   }
 
   .sky-search-btn-open-applied {
     &.sky-search-btn-open,
     &.sky-search-btn-open:hover {
-      color: $sky-highlight-color-success;
-      border: 2px solid $sky-highlight-color-success;
-      padding: 5px 11px;
+      background-color: var(
+        --sky-override-search-applied-background-color,
+        var(--sky-color-background-selected-soft)
+      );
+      border: var(--sky-override-search-applied-border, none);
+      box-shadow: var(
+        --sky-override-search-applied-box-shadow,
+        inset 0 0 0
+          var(
+            --sky-override-search-appied-border-width,
+            var(--sky-border-width-action-selected-s)
+          )
+          var(
+            --sky-override-search-applied-color,
+            var(--sky-color-border-selected)
+          )
+      );
+      color: var(
+        --sky-override-search-applied-color,
+        var(--sky-color-icon-action)
+      );
+      padding: var(
+        --sky-override-search-mobile-button-padding,
+        var(--sky-space-inset-balanced-s)
```

Comment:
The spec seems to call for `space.inset.pillarbox.1_2.{r,l}.m` and `space.inset.pillarbox.1_2.{t,b}.m` not `space.inset.balanced.s`. Is the current token correct?